### DISCUSSION TITLE
Triple-hop or allow connection to be routed through as many servers as possible

### DISCUSSION THREAD

**Discussion by AltoidUC** [mullvad/mullvadvpn-app#9058] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

If it's possible, I'd certainly like to see a new function for multi-hop, allowing you to select 3 individual servers, or maybe even as much as possible. This idea came from the idea of onion routing that routes your connection through multiple servers. For example, Tor Project has an entry, middle, and exit.

### Alternative solutions

None that I can think of, just multi-hop with 2 servers. I'd think the idea would be pretty simple to implement, especially considering there is already a base for multi-hop.

### Type of feature

- [X] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [ ] Easier to use
- [ ] Other

### Operating System

- [ ] Android
- [ ] iOS
- [X] Windows
- [ ] macOS
- [X] Linux

**Comment by MarkusPettersson98** [comment]
Hi, thanks for the feature suggestion! :yellow_heart:  I will lift this internally.

**Comment by MarkusPettersson98** [comment]
Hi again, we have decided to not implement this for now. Keeping the issue open for now to track the engagement from the community, but do not expect any updates from us Mullvad devs on this feature request in a while. Thanks for your understanding :yellow_heart:

**Comment by AltoidUC** [comment]
> Hi again, we have decided to not implement this for now. Keeping the issue open for now to track the engagement from the community, but do not expect any updates from us Mullvad devs on this feature request in a while. Thanks for your understanding 💛

That's a shame, I was hoping for something like this, but there probably won't be very much demand for it. Still hoping that one day it'll be implemented though. :)

**Comment by ave9858** [comment]
> For example, Tor Project has an entry, middle, and exit.

This is mainly because Tor relays are not fully trusted, so multiple hops divides the information each relay has so that none can fully identify you. With Mullvad, it makes more sense to either trust all of the servers or none of them, so onion routing isn't really helpful since the Mullvad server network isn't very diverse
